packages/experiments/src/components/Announced/examples/Announced.SearchResults.Example.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import { Announced } from '../Announced';
+import { VerticalStack } from '../../Stack';
+import { TagPicker } from 'office-ui-fabric-react/lib/Pickers';
+
+const _testTags = [
+  'black',
+  'blue',
+  'brown',
+  'cyan',
+  'green',
+  'magenta',
+  'mauve',
+  'orange',
+  'pink',
+  'purple',
+  'red',
+  'rose',
+  'violet',
+  'white',
+  'yellow'
+].map(item => ({ key: item, name: item }));
+
+export interface IAnnouncedSearchResultsExampleState {
+  seconds: number;
+  numberOfSuggestions: number;
+}
+
+export interface IAnnouncedSearchResultsExampleProps {}
+
+export class AnnouncedSearchResultsExample extends React.Component<
+  IAnnouncedSearchResultsExampleProps,
+  IAnnouncedSearchResultsExampleState
+> {
+  private timer: number;
+
+  constructor(props: {}) {
+    super(props);
+    this.state = {
+      seconds: 0,
+      numberOfSuggestions: 0
+    };
+
+    this.timer = setInterval(() => {
+      this.setState({ seconds: this.state.seconds + 1 });
+    }, 1000);
+  }
+
+  public render(): JSX.Element {
+    return (
+      <div>
```

Comment:
nit: you can return a [React.Fragment](https://reactjs.org/docs/fragments.html) here `<></>` or `<React.Fragment></React.Fragment>` since these are simply wrapper DIVs.